### DISCUSSION TITLE
posts/gatus-uptime-monitoring/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#333] [Comments]
# posts/gatus-uptime-monitoring/

Meet Gatus a self-hosted, open source, health dashboard that lets you monitor all of you services and systems! This dashboard not only tracks your uptime, but also measure the results plotting the results on a chart over time. It also hooks into systems like Slack, Team, Discord, Twilio, and more! Join me as we configure and deploy Gatus into our own environment to measure and monitor all the things!

https://technotim.live/posts/gatus-uptime-monitoring/

**Comment by mazarian** [comment]
Thank you for the guide! When using postgres, how can we make sure the database doesn't grow out of control? Unlike Uptime Kuma, Gatus doesn't really specify how long data is stored for, so I assume that is something we have to take care of on the backend?
